openvdb/openvdb/tools/LevelSetRebuild.h

```diff
@@ -250,13 +265,36 @@ doLevelSetRebuild(const GridType& grid, typename GridType::ValueType iso,
 
     QuadAndTriangleDataAdapter<Vec3s, Vec4I> mesh(points, primitives);
 
+#if OPENVDB_USE_ORACLE_IN_REBUILD
+    auto backToOldGrid = [&xform, &grid](const Coord& coord) -> openvdb::math::Vec3d {
+        return grid.transform().worldToIndex(xform->indexToWorld(coord));
+    };
+
+    auto interiorTest = [acc = grid.getConstAccessor(), &backToOldGrid, &xform](const Coord& coord) -> bool {
```

Comment:
This looks sub-optimal to me; a ValueAccessor is created, which is only used once. Can this be improved or is it not a performance concern? Or is the sampler using it for all the stencil points?

Reply:
If I understand properly, the 
[acc = grid.getConstAccessor, ... ]
will add a new by-value const accessor to the capture of the interiorTest lambda.  This is done at the time of creating this lambda, so this is not created on each call to the lambda, but instead on the creation.
Likewise, the interiorTest doesn't have to be thread safe so long as it is copy-constructable and copy-constructed versions of it can run in separate threads.   So we should be doing copy-construction of the accessor before going to separate threads.